packages/teleport/src/components/hooks/useUrlFiltering/labelClick.ts

```diff
@@ -1,5 +1,21 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 import { AgentLabel } from 'teleport/services/agents';
-import { ResourceUrlQueryParams } from './getUrlQueryParams';
+import { ResourceUrlQueryParams } from './getQueryParams';
 import encodeUrlQueryParams from './encodeUrlQueryParams';
 
 export default function labelClick(
```

Comment:
it seems `labelClick.ts` is only used in `useUrlFiltering.ts`, i would move it there and delete this file